test/end-to-end/helpers/JSWait.js

```diff
@@ -14,13 +14,13 @@ class JSWait extends codecept_helper {
         const helper = this.helpers.WebDriver || this.helpers.Puppeteer;
         const helperIsPuppeteer = this.helpers.Puppeteer;
 
-        helper.click(text, locator).catch(err => {
-            console.error(err.message);
-        });
-
         if (helperIsPuppeteer) {
+            helper.click(text, locator).catch(err => {
```

Comment:
best to use await and a try/catch block here
so, move this before the if:
try {
await helper.click(text, locator);
} catch (err) {
console.error(err.message);
}

Reply:
good thought, will add that change

Reply:
done